### PR TITLE
fixing icedata tests

### PR DESCRIPTION
**Pull request by FraPochetti**

### CODE REVIEW

File: icedata/datasets/pets/parser.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def labels(self, o) -> List[Hashable]:
         # there is an image with two cats (same breed)
         num_objs = len(self._root.findall("object"))
 
-        return [class_id] * num_objs
```
Comment (lgvaz) [inline icedata/datasets/pets/parser.py:39]: the line that gets `class_id` can also be removed

**Review by lgvaz — COMMENTED** [review]
small suggestion =)

File: icedata/datasets/pets/parser.py (hunk 1 above)
Comment (FraPochetti) [reply icedata/datasets/pets/parser.py:39]: good point!
Done!

**Review by lgvaz — APPROVED** [review]
alright, looks good!

thanks!!

**Pull request merged**